src/main/resources/liquibase/crop_changelog/12_5_0.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="mderamos" id="v12.5.0-1">
+		<preConditions onFail="MARK_RAN">
+			<and>
+				<columnExists tableName="key_sequence_register" columnName="suffix"/>
+				<not>
+					<sqlCheck expectedResult="0">
+						SELECT COUNT(*) FROM key_sequence_register duplicate, key_sequence_register ksr
+							WHERE ksr.key_prefix = duplicate.key_prefix
+							AND ksr.last_used_sequence > duplicate.last_used_sequence;
+					</sqlCheck>
+				</not>
+			</and>
+		</preConditions>
+		<comment>
+			Remove duplicate entries from key_sequence_register
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			DELETE duplicate FROM key_sequence_register duplicate, key_sequence_register ksr
```

Comment:
We need to handle cases of duplicate prefixes with the same `last_used_sequence `value